TypeCobol/Compiler/Nodes/Node.cs

```diff
@@ -821,6 +821,7 @@ private void Dump(StringBuilder str, int i)
         /// <returns>Correpsonding DataDefinition</returns>
         public DataDefinition GetDataDefinitionFromStorageAreaDictionary(StorageArea searchedStorageArea, bool? isReadDataDefiniton=null)
```

Comment:
Can you add attribute `[CanBeNull]` before StorageArea searchedStorageArea ?
This way Resharper will know that the value can be null.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1595/commits/59139c835bd4751fb47886a0d3305206f1272b62